src/components/tipRecords/TipPreview.vue

```diff
@@ -388,28 +388,20 @@ export default {
 
 ::v-deep .tip__cover-preview {
   position: relative;
-  padding-top: 55%;
   overflow: hidden;
   width: 100%;
+  height: 319px;
   max-width: 100%;
   background: #000;
   border-radius: 0.5rem 0.5rem 0 0;
 
   img {
-    bottom: 0;
-    display: block;
-    left: 0;
-    margin: auto;
     max-width: 100%;
     width: 100%;
-    position: absolute;
-    right: 0;
-    top: 0;
     border: none;
     height: auto;
```

Comment:
```suggestion
    height: 319px; //?
    object-fit: cover/contain;
```
instead of absolute positioning